akka-http-core/src/main/java/akka/http/javadsl/model/MediaTypes.java

```diff
@@ -187,6 +187,119 @@ private MediaTypes() { }
     public static final MediaType.Binary VIDEO_X_SGI_MOVIE = akka.http.scaladsl.model.MediaTypes.video$divx$minussgi$minusmovie();
     public static final MediaType.Binary VIDEO_WEBM = akka.http.scaladsl.model.MediaTypes.video$divwebm();
 
+    public static MediaType.Binary applicationBinary(String subType, boolean compressible, String... fileExtensions) {
+        akka.http.scaladsl.model.MediaType.Compressibility comp = compressible ?
```

Comment:
`Compressibility` - using the same trick we have in most javadsl - empty interface in javadsl which this one extends.
I'll take this over